src/Smoothie.js

```diff
@@ -1,15 +1,37 @@
-import React, { Component } from 'react'
+import React, { Component } from 'react';
+import './Smoothie.css';
 
 export default class Smoothie extends Component {
   constructor(props) {
     super(props)
+    this.state = {
+      isSelected: false
+    }
   }
 
+  expandRecipe = () => {
+    const {isSelected} = this.state
+    const {showRecipe} = this.props
```

Comment:
Nice use of ES6 destructuring, you'll see this all over the place in React apps.